lib/broccoli/ember-app.js

```diff
@@ -1275,17 +1251,21 @@ EmberApp.prototype.javascript = function() {
   }
 
   this.import('vendor/ember-cli/vendor-prefix.js', {prepend: true});
-  this.import('vendor/addons.js');
-  this.import('vendor/ember-cli/vendor-suffix.js');
```

Comment:
and if you add a setter to `_scriptOutputFiles` or overwrite `import`, seems fine? It's your own foot and you should have known better?